# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/NamespaceImports.h
# Commit: 23cc649c6e14
# Full Hash: 23cc649c6e14647072116e6d2396cb615996aa61
# Author: Steve Fink <sfink@mozilla.com>
# Date: 2021-08-13 09:27:46
# Regressor Bug: 1720422
# File Overlap Count: 1
# Description:
#   Bug 1720422 - Move js::Scalar -> JS::Scalar for public API, but alias it as js::Scalar r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D120292
# ==============================================================================

diff -r 16695064a637 -r 23cc649c6e14 js/src/NamespaceImports.h
--- a/js/src/NamespaceImports.h	Thu Aug 12 20:44:17 2021 +0000
+++ b/js/src/NamespaceImports.h	Thu Aug 12 20:44:18 2021 +0000
@@ -32,11 +32,15 @@
 
 class JS_PUBLIC_API PropertyDescriptor;
 
+namespace Scalar {}
+
 }  // namespace JS
 
 // Do the importing.
 namespace js {
 
+namespace Scalar = JS::Scalar;
+
 using JS::BooleanValue;
 using JS::DoubleValue;
 using JS::Float32Value;